### PR TITLE
[SPARK-10669] [Docs] Link to each language's API in codetabs in ML docs: spark.mllib

### PR DESCRIPTION
**Pull request by keypointt**
In the Markdown docs for the spark.mllib Programming Guide, we have code examples with codetabs for each language. We should link to each language's API docs within the corresponding codetab, but we are inconsistent about this. For an example of what we want to do, see the "ChiSqSelector" section in https://github.com/apache/spark/blob/64743870f23bffb8d96dcc8a0181c1452782a151/docs/mllib-feature-extraction.md
This JIRA is just for spark.mllib, not spark.ml.

Please let me know if more work is needed, thanks a lot.

### CODE REVIEW

File: docs/mllib-frequent-pattern-mining.md [hunk 1]
```diff
@@ -195,10 +202,12 @@ results.collect().foreach { rule =>
 </div>
 
 <div data-lang="java" markdown="1">
-[AssociationRules](api/java/org/apache/spark/mllib/fpm/AssociationRules.html)
+[AssociationRules](api/scala/index.html#org.apache.spark.mllib.fpm.AssociationRules)
```
Comment (srowen) [inline docs/mllib-frequent-pattern-mining.md]: Unintentional change? I think this was correct before

File: docs/mllib-linear-methods.md [hunk 2]
```diff
@@ -230,6 +232,8 @@ Spark Java API uses a separate `JavaRDD` class. You can convert a Java RDD to a
 calling `.rdd()` on your `JavaRDD` object. A self-contained application example
 that is equivalent to the provided example in Scala is given bellow:
 
+Refer to the [`BinaryClassificationMetrics` Java docs](api/java/org/apache/spark/mllib/evaluation/BinaryClassificationMetrics.html) for details on the API.
```
Comment (srowen) [inline docs/mllib-linear-methods.md]: This doesn't parallel the Scala note above with SVMWithSGD?

File: docs/mllib-statistics.md [hunk 3]
```diff
@@ -86,6 +90,8 @@ System.out.println(summary.numNonzeros()); // number of nonzeros in each column
 which contains the column-wise max, min, mean, variance, and number of nonzeros, as well as the
 total count.
 
+Refer to the [`Statistics` Python docs](api/python/pyspark.mllib.html#pyspark.mllib.stat.Statistics) for more details on the API.
```
Comment (srowen) [inline docs/mllib-statistics.md]: Again should this be about MultivariateStatisticalSummary ?

**Comment by srowen** [comment]
Looks generally OK to me, with a few questions. You might also build the docs in docs/ with `SKIP_API=1 jekyll build` and spot-check your changes to make sure they look as you intended.

**Comment by keypointt** [comment]
@srowen thank you for the code review, I'll change the code lines you mentioned above and test it out. :)

File: docs/mllib-linear-methods.md (hunk 2 above)
Comment (keypointt) [inline docs/mllib-linear-methods.md]: you are right, I'll correct it

File: docs/mllib-frequent-pattern-mining.md (hunk 1 above)
Comment (keypointt) [inline docs/mllib-frequent-pattern-mining.md]: I'll correct it

File: docs/mllib-statistics.md (hunk 3 above)
Comment (keypointt) [inline docs/mllib-statistics.md]: I'll correct it

**Comment by keypointt** [comment]
Hi @srowen I've corrected what you mentioned, and added markdown level for the divs missing markdown="1". I've run "jekyll build" and tested on my local machine too. Thanks a lot :)

**Pull request closed without merging**